src/sentry/static/sentry/app/utils/projects.tsx

```diff
@@ -411,6 +415,7 @@ async function fetchProjects(
     cursor?: typeof cursor;
     per_page?: number;
     all_projects?: number;
+    get_counts?: number;
```

Comment:
Is this used?

Reply:
Missed this during an earlier change, :+1: removed